packaging/images/openshift-ci/rpm.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM registry.redhat.io/ubi8/ubi
+
+ENV GOPATH=/go
+
+WORKDIR $GOPATH/src/github.com/openshift/microshift
+
+RUN dnf update -y &&  \
+    dnf install --setopt=tsflags=nodocs -y \
+    git \
+    golang-1.17.12-1.module+el8.6.0+16014+a372c00b && \
```

Comment:
Is there some way to base this image on a golang builder image so we don't have to keep updating the version here for every release?